src/cpp/session/SessionHttpMethods.cpp

```diff
@@ -221,6 +218,7 @@ Error startHttpConnectionListener()
       // set the standalone port so rpostback and others know how to
       // connect back into the session process
       std::string port = safe_convert::numberToString(endpoint.port());
+      auto rule = core::http::createNoProxyRule("127.0.0.1", port);
```

Comment:
did you need 
```suggestion
      auto rule = core::http::createNoProxyRule("127.0.0.1", port);
      core::http::proxyUtils().addNoProxyRule(std::move(rule));
```
here as well?

Reply:
Also, is it possible for `startHttpConnectionListener` to get called more than once?

Reply:
Thanks! I'll fix that up.

I don't think it's possible to get called more than once here, but I think this might already be superfluous -- since we're already adding these no-proxy rules earlier in SessionMain. I'm going to remove it and see...